scripts/build-webpack.js

```diff
@@ -33,7 +36,7 @@ const compiler = webpack({
     }),
   ],
   output: {
-    filename: `yarn-${version}.js`,
+    filename: `[name]-${version}.js`,
```

Comment:
Personally, I'd use the same version number everywhere (whether changes were made or not) because I think that is the clearest and easiest system to understand. I think Babel does this, and Lerna calls this the "synchronized" versioning scheme and is used by default. With the synchronized scheme it would be simple to publish a new lockfile package every time you publish a new yarn package and they could be part of the same build and release script.

Versioning the packages independently has advantages too... but those advantages are usually for the consumer (less frequent updates) and creates more work for the developers. Deciding on the version number of the lockfile package and whether or not to publish a new version becomes one more manual task that could be forgotten instead of part of the regular release process.